fenix/app/src/main/java/org/mozilla/fenix/library/history/HistoryFragment.kt

```diff
@@ -110,6 +118,7 @@ class HistoryFragment : LibraryPageFragment<History>(), UserInteractionHandler,
                         HistoryNavigationMiddleware(
                             navController = findNavController(),
                             openToBrowser = ::openItem,
+                            onBackPressed = { requireActivity().onBackPressedDispatcher.onBackPressed() },
```

Comment:
Very nice 😎 I snuck in a similar update for the only other lambda that was written like this in the middleware definitions

Reply:
Just to close the loop, I reverted the similar update because the function reference syntax causes an access of `historyView` before it is initialized. Wrapping the access in a lambda (as it was before) means the object is accessed lazily and has time to be initialized.